uraniborg/AndroidStudioProject/Hubble/app/src/main/java/com/uraniborg/hubble/MainActivity.java

```diff
@@ -44,7 +44,7 @@
 
 public class MainActivity extends AppCompatActivity {
   final String TAG = "HUBBLE";
-  private final String VERSION = "1.2.0";   // NOTE: to be updated for every Hubble release
+  private final String VERSION = "1.3.0";   // NOTE: to be updated for every Hubble release
```

Comment:
Instead of manually updating this value on each release, should you use something like gradle's `BuildConfig .VERSION_NAME` to keep it consistent?

Reply:
That's a good suggestion. Let me fix that.